.ci/docker/ci_commit_pins/triton-xpu.txt

```diff
@@ -1 +1 @@
-cfb7d5314748542fed42d0837bef1a6d177da2d6
+83111ab22be6e4a588d184ac45175986a7dde9fcgi
```

Comment:
this should be `83111ab22be6e4a588d184ac45175986a7dde9fc` -- looks like it has two extra chars at the end